matrix-sdk/src/main/java/org/matrix/androidsdk/data/store/MXFileStore.java

```diff
@@ -279,6 +282,11 @@ private void setIsKilled(boolean isKilled) {
         }
     }
 
+    public MXFileStore setMetricsListener(MetricsListener metricsListener) {
+        this.mMetricsListener = metricsListener;
+        return this;
```

Comment:
Only this one return `this`. Consider also for the other similar method, or don't do that